plugins/source/slack/Makefile

```
@@ -0,0 +1,40 @@
+# Test unit
+.PHONY: test
+test:
+	go test -timeout 3m ./...
+
+# Install tools
+.PHONY: install-tools
+install-tools:
+	@echo Installing tools from tools/tool.go
+	@cat tools/tool.go | grep _ | awk -F'"' '{print $$2}' | xargs -tI % go install %
+
+# Install pre-commit hooks. This requires pre-commit to be installed (https://pre-commit.com/)
+.PHONY: install-hooks
+install-hooks:
+	pre-commit install
+
+.PHONY: lint
+lint:
+	@golangci-lint run --timeout 10m --verbose
```

```suggestion
	golangci-lint run --config ../../.golangci.yml
```

See https://github.com/cloudquery/cloudquery/pull/5562